frontend/ui/src/components/AiryWebsocket/index.tsx

```diff
@@ -28,7 +29,7 @@ const mapDispatchToProps = dispatch => ({
   addMessages: (conversationId: string, messages: Message[]) => dispatch(addMessagesAction({conversationId, messages})),
   onChannel: (channel: Channel) => dispatch(setChannelAction(channel)),
   getConversationInfo: (conversationId: string) => dispatch(getConversationInfo(conversationId)),
-  onMetadata: (metadataEvent: MetadataEvent) => dispatch(setMetadataAction(metadataEvent)),
+  onMetadata: (metadataEvent: MetadataEvent) => dispatch(setMetadataAction(camelcaseKeys(metadataEvent, {deep: true}))),
```

Comment:
This is missing the stop path for `user_data` that @AudreyKj implemented in other parts where we map metadata. It's needed, because fields on `metadata.user_data `are set by the user so they don't conform to our casing